### PR TITLE
Happy-API

### PR DESCRIPTION
**Pull request by lilihorvath**

### CODE REVIEW

**Comment by Snabbina** [comment]
Hey Lili, hope you are good! Im supposed to review your project, but the Heroku link does not seem to work... de deployed link should look something like this "https://name-oftheproject-api.herokuapp.com/" and plz also add the deployed link to the frontend so I can check that out too for you! /Have a nice evening! /S

File: server.js [hunk 1]
```diff
@@ -7,23 +7,63 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
+  createdAt: {
+    type: Date,
+    default: Date.now
+  }
 })
 
-// Start the server
-app.listen(port, () => {
-  console.log(`Server running on http://localhost:${port}`)
+app.get('/', async (req, res) => {
```
Comment (KarolCislowski) [inline server.js:37]: this endpoint should be `GET /thoughts` 😸

File: server.js [hunk 2]
```diff
@@ -7,23 +7,63 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
+  const thoughts = await Thought.find().sort({createdAt: 'desc'}).limit(20).exec()
+  res.json(thoughts)
 })
+
+app.post('/', async (req, res) => {
```
Comment (KarolCislowski) [inline server.js:42]: this endpoint should be `POST /thoughts` 😸

File: server.js [hunk 3]
```diff
@@ -7,23 +7,63 @@ const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
...
+  message: {
+    type: String,
+    required: true, 
+    minLength: 5,
+    maxLength: 140,
+  },
+  username: {
+    type: String, 
+    maxLength: 20,
+  },
```
Comment (KarolCislowski) [inline server.js:26]: you created this field in the model, but you never used it. why? 🤔

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Please change wrong endpoints and add a link to the hosted frontend. 😺

**Pull request closed without merging**